src/ZoneServer/Skills/Handlers/Swordsman/Doppelsoeldner/Punish.cs

```diff
@@ -23,7 +23,7 @@ public class Doppelsoeldner_Punish : IGroundSkillHandler
 		private const float MaxTargetDistance = 30f;
 		private const float MaxMoveDistance = 140f;
 		private const float KnockdownMultiplier = 1.5f;
-		private const float HealDebuffPerLevel = 33f;
+		private const float HealDebuffPerLevel = 3.3f;
```

Comment:
It seems like you changed code that was deliberately done this way because the client needed the given values. Are the displayed information on the buff tooltips still correct after these changes?

Reply:
![image](https://github.com/user-attachments/assets/b9a81438-bf60-4780-8f4f-b0103d0b6628)

Yeah this variable doesn't affect the tool tip values.

Reply:
It's actually not the tooltip for the skill, it's the mouseover on the debuff in the victim's debuff window.  Try inflicting it on a mob, then mouse over the debuff near the middle of the screen.

Reply:
It displays the debuff based on the value of the buff's numarg2, which is divided by 1000.  So to get 3.3%, you have to pass in 3300 to the numarg2.

Reply:
![image](https://github.com/user-attachments/assets/bb739eff-fae9-4d84-a6ec-0f8f502cbc97)
Fixed!